src/hooks/use-chat-context.tsx

```diff
@@ -120,7 +120,9 @@ export const ChatProvider: React.FC< ChatProviderProps > = ( { children } ) => {
 			! initialLoad[ selectedSite.id ] &&
 			isCurrent &&
 			! pluginsList[ selectedSite.id ] &&
-			! themesList[ selectedSite.id ]
+			! themesList[ selectedSite.id ] &&
+			selectedSite.running === true &&
```

Comment:
Should we limit this to running sites? If not, the AI chat context won't retrieve the themes and plugins for stopped sites when selecting them. It's true that for the case of the focus listener, we do limit it to sites running. But that's because we only expect running sites to modify the themes/plugins list ([discussion](https://github.com/Automattic/studio/pull/252/files#r1645759442)).

Reply:
I tried removing the condition `selectedSite.running === true` and, although it works to fetch themes/plugins when selecting already-created sites, it doesn't retrieve properly newly created sites as it's returning empty arrays.

We might need to tweak this logic to ensure we fetch the themes/plugins for the following cases:
1. After creating a new site.
2. Selecting a site for the first time in a session (i.e. we didn't perform an initial load).

Reply:
@fluiddot , I updated the logic. It should be ok now. What do you think? Do you find any cases that this doesn't produce the expected behavior?